Automattic/wp-calypso
# Badge: Update references to point to new package component

Related to #77564

## Proposed Changes

* Following up on #78809 and #79303, this repoints any references to `Badge` from Calypso components to `@automattic/components`
* I recognize this is a sizable change that touches many files. We effectively made this change in the background with #79303, so all Badge components are running the new/relocated code already. This PR repoints the reference from the wrapper to the "real" code, so I don't expect this to cause any issues, and I've spot-checked as many instances as I could find on the front end.

## Testing Instructions



* Make sure tests pass.
* Do a spot-check of Badge instances to ensure they still look/work as expected.

## Pre-merge Checklist
